### PR TITLE
InfluxDB: Deprecate direct browser access in data source

### PR DESCRIPTION
**Pull request by gabor**
In the InfluxDB datasource it is possible to choose browser-access, meaning queries go directly from the browser to the influxdb server. We are going to remove this possibility in a future release, so for now we will mark it as deprecated and show a deprecation-warning.

### CODE REVIEW

**Review by Elfo404 — APPROVED** [review]

File: docs/sources/datasources/influxdb/_index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Name        | Description
 `Name`      | The data source name. This is how you refer to the data source in panels and queries. We recommend something like `InfluxDB-InfluxQL`.
 `Default`   | Default data source means that it will be pre-selected for new panels.
 `URL`       | The HTTP protocol, IP address and port of your InfluxDB API. InfluxDB API port is by default 8086.
-`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser.
+`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser (**WARNING**: Browser access is deprecated and will be removed in a future release).
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/influxdb/_index.md:35]:
```suggestion
`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser.
**Note**: Browser access is deprecated and will be removed in a future release.
```

Comment (achatterjee-grafana) [reply docs/sources/datasources/influxdb/_index.md:35]: It would be better to add this as a note, instead of a warning.

**Review by achatterjee-grafana — COMMENTED** [review]
Added a minor change suggestion.

File: docs/sources/datasources/influxdb/_index.md (hunk 1 above)
Comment (gabor) [reply docs/sources/datasources/influxdb/_index.md]: @achatterjee-grafana thanks, applied!

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**